src/services/BudgetService.ts

```diff
@@ -130,4 +150,40 @@ export class BudgetService {
       )
     }
   }
+
+  static getProposalsBudgetingMinAndMaxDates(proposals: Pick<ProposalAttributes, 'start_at'>[]): {
```

Comment:
I think we can drop `budgeting` from the name function 🤔

Reply:
Yeah, the thing is that it won't just return you the min and max dates for every proposal, it filters those that started after the budgeting system started. I thought the name should give you a hint